pkgs/by-name/sm/smtp4dev/package.nix

```diff
@@ -0,0 +1,59 @@
+{ lib, buildDotnetModule, fetchFromGitHub, dotnetCorePackages, buildNpmPackage }:
+let
+
+  version = "3.3.4";
+  gsrc = fetchFromGitHub {
+    owner = "rnwood";
+    repo = "smtp4dev";
+    rev = version;
+    hash = "sha256-ARq5OpFJ4o9KdBXvzOx7QLB8GNfmXWjO0RR4jKP8qRI=";
+  };
+
+  webgui = buildNpmPackage rec {
+    pname = "smtp4dev-webgui";
+    inherit version;
+
+    src = gsrc + "/Rnwood.Smtp4dev/ClientApp";
+
+    npmDepsHash = "sha256-VBcfRKYe/uPf6urWuLI5TrnX9bgiKiZKo+N4zL7O3SM=";
+    postInstall = ''
+      rm -rf $out/*
+      mkdir -p $out
+      cp  -R ../wwwroot $out/wwwroot
+    '';
+  };
+in
+buildDotnetModule rec {
+  pname = "smtp4dev";
+  src = gsrc;
+  inherit version;
```

Comment:
```suggestion
  inherit version src;
```

Reply:
Due to the previous reason it's not possible.

Reply:
Wow I'm dumb! Thanks ❤️